src/main/java/com/mixpanel/android/viewcrawler/ViewVisitor.java

```diff
@@ -282,6 +251,51 @@ private boolean verifyLayout(View target, Set<Integer> rules) {
                 }
             }
 
+            int size = mArgs.size();
+            for (int i = 0; i < size; i++) {
+                LayoutRule layoutRule = mArgs.get(i);
+                final int viewId = layoutRule.viewId;
+                final int newVerb = layoutRule.verb;
+                final int newAnchorId = layoutRule.anchor;
+                final View currentNode = idToChild.get(viewId);
+
+                // typecast ViewGroup.LayoutParams is needed for API 18 and lower
```

Comment:
I'll remove this.. This is just a stupid Android thing. You have NO way to make a copy of RelativeLayout.LayoutParams.. I wanted to have RelativeLayout.LayoutParams as the original value so that I can simply call view.setParams in the cleanup to revert the changes. But you can only get a reference to it.